server/config/routes.rb

```diff
@@ -250,6 +256,7 @@
     jsonapi_resources :franchises
     jsonapi_resources :installments
     jsonapi_resources :mappings
+    jsonapi_resources :list
```

Comment:
Thanks, I'm not used to jsonapi resources yet, so I'm probably going to make a lot of those little mistakes >_<